Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM python:2
```

Comment:
At some point, this really needs to be updated to Python 3, since Python 2 will no longer be supported from next year.

I saw you were doing this with some of the drivers, but just making a note here 👍

Reply:
yep, we're all waiting for a py3 version of web.py - which hopefully will be available in the coming months